IndividualSpreadFooting.tex

```diff
@@ -213,14 +213,14 @@ \section{Design for Perimeter Shear} Depth of footing is fixed from
 develop thumb rules for fixing depth of footings. It may be noted that
 there is no dire need of exactness in fixing the value of depth of
 footing, only it should be more than adequate for the actions imposed
-on a footing. \tablemacro \ref{chaptertable}, based on
-\equmacro (\ref{eq:areafck}), is developed for footings of Types ($a$) 
-and ($b$) (\figmacro \ref{Types-of-individual-spread-footing}).It gives
+on a footing. \tablem \ref{chaptertable}, based on
+\eqn (\ref{eq:areafck}), is developed for footings of Types ($a$) 
+and ($b$) (\fig \ref{Types-of-individual-spread-footing}).It gives
 values of $D/A$ for various practicable values of $p$. It is seen that,
 for safety in beam shear, these values are to be increased by 10\% in 
-case of steel types \Fefouronefivemacro and \Fefivezerozeromacro. For 
+case of steel types \fefouronefive and \fefivezerozero. For 
```

Comment:
Mandeep, As I said earlier, We can't use numerics in Macroname. These name \fefouronefive and so on are correct.